### DISCUSSION TITLE
Wsparcie dla czujnika HDC1080?

### DISCUSSION THREAD

**Discussion by tnowak** [hackerspace-silesia/Smogomierz#124] [Ideas]
Z testu poniżej wynika, że czujnik HDC1080 w zakresie pomiaru temperatury oraz wilgotności jest nawet dokładniejszy niż BME280:
https://www.youtube.com/watch?v=ynNeiFEJnrA
Czy planujecie dopisanie wsparcia dla niego?

**Comment by bfaliszek** [comment]
Jak tylko znajdę czas. Mam nawet kilka do testu.

**Comment by tnowak** [comment]
Dzięki! Napisz proszę tu, gdy się uda
